docs/src/getting-started/quickstart.md

```diff
@@ -208,6 +208,14 @@ If all goes well, you should see the following:
 ▪▪▪▪▪ ✅ Successfully deployed indexer.
 ```
 
+> #### What is a "deployment" exactly?
+>
+> A _deployment_ within the context of Fuel's indexer is a series of steps taken to get your indexer project running in the wild.
+>
+> These series of steps involves compiling your indexer project to a `wasm32-unknown-unknown` target, uploading the indexer to a running Fuel indexer service, and having that service register an executor, and build database tables for this indexer. These series of steps, by definition, constitute a ["deployment"](https://en.wikipedia.org/wiki/Software_deployment).
```

Comment:
Consider:
> This series of steps involves compiling your indexer project to a `wasm32-unknown-unknown` target and uploading the indexer to a running Fuel indexer service. The service will then register an executor and build database tables for this indexer. Once this series of steps has completed, your indexer is considered to be "deployed".

Adjusted sentence flow and fixed singular vs plural series.

Reply:
Will keep the "this series" since we allude to the "series of steps" at the beginning of the blurb.